functions.js

```diff
@@ -0,0 +1,197 @@
+/* ## Question 1
+
+Write a function that returns the largest of two numbers. If they are equal, return the second one.
+
+Example:
+
+Input: `12, 4`
+
+Output: `12`
+*/
+console.log('#1');
+
+function twoLargestNum(numbers) {
+  numbers.sort(function(a,b){
+    return b - a;
+  });
+  return numbers[0] + numbers[1]
+}
+let numbers = twoLargestNum([10,20,30,40,50])
+console.log(numbers);
+
```

Comment:
We have to understand the question. With this problem it should take two numerical arguments and return which number is larger.  

With this clarification, Im positive that you can create a new function that will excute this